syntax = "proto3";
message MyVarint
{
	int32 num1 = 1;
	int64 num2 = 2;
	bool num3 = 3;
}

message MyFixed
{
	fixed32 num1 = 1;
	fixed64 num2 = 2;
}

message MyLen
{
	string num1 =1;
}

message MyCustom
{
	repeated int32 num1 =1;
	map<int32,string> num2 =2;
}